src/main/java/org/spongepowered/common/world/SpongeExplosionBuilder.java

```diff
@@ -47,6 +50,7 @@
     private int resolution = 16;
     private float randomness = 1.0F;
     private double knockback = 1;
+    private Optional<ResistanceCalculator> resistanceCalculator = Optional.empty();
```

Comment:
Never _store_ an `Optional`, this should be a `@Nullable` field.

Note, you were actually leading up to an NPE if someone `reset()` then `build()` the builder.